### ISSUE TITLE
Complie error because streamSkipUntil() is a protected constuctor

### ISSUE THREAD

**Issue by u4mzu4** [vshymanskyy/TinyGSM#456]
[X ] I have read the Troubleshooting section of the ReadMe

## What type of issues is this?

[ ] Request to support a new module
    
[ X] Bug or problem compiling the library
[ ] Bug or issue with library functionality (ie, sending data over TCP/IP)
[ ] Question or request for help

### What are you working with?

Modem: 
Main processor board: 
TinyGSM version: 
Code: 

modem.sendAT(GF("+FTPPUT=1"));
response = modem.waitResponse();
delay(2500);
modem.streamSkipUntil(',');
modem.streamSkipUntil(',');
lengthOfDataStr = modem.stream.readStringUntil('\n');
lengthOfDataInt = lengthOfDataStr.toInt();

**Comment by gasagna** [comment]
I agree that this limits the use of the modem for sending other AT commands than those that have already been implemented. Other functions such as `streamGetIntBefore` or `streamGetFloatBefore` are really useful in custom code to parse the response to some AT commands, but we can't really use them.

Can these functions be made part of the `public` interface? What is the reason for them to be marked as `protected`?
